tools/spectral/ipa/rulesets/IPA-108.yaml

```
@@ -3,31 +3,60 @@
 
 rules:
   xgen-IPA-108-delete-response-should-be-empty:
-    description: Delete method response should not have schema reference to object.
+    description: |
+      Delete method response should not have schema reference to object.
+
+      ##### Implementation details
+      Rule checks for the following conditions:
+        - Applies to 204 responses in DELETE methods
+        - Verifies that the response does not contain a schema property
+        - Fails if any content type in the response has a defined schema as reference
     message: '{{error}} https://mdb.link/mongodb-atlas-openapi-validation#xgen-IPA-108-delete-response-should-be-empty'
     severity: warn
     given: $.paths[*].delete.responses[204]
     then:
       function: IPA108DeleteMethodResponseShouldNotHaveSchema
 
   xgen-IPA-108-delete-method-return-204-response:
-    description: DELETE method must return 204 No Content.
+    description: |
+      DELETE method must return 204 No Content.
+
+      ##### Implementation details
+      Rule checks for the following conditions:
+        - Applies to all DELETE methods
+        - Verifies that the method includes a 204 response code
```

[Nit] To be consistent with others
```suggestion
         - Verifies the 204 No Content response code is present
        - Fails if the method lacks a 204 No Content response or defines a different 2xx status code
```

---

yes. Separately WDYT about GET vs Get operations? We can align it in follow up